plugins/signals-backend/src/plugin.ts

```diff
@@ -58,7 +58,7 @@ export const signalsPlugin = createBackendPlugin({
           }),
         );
         httpRouter.addAuthPolicy({
-          path: '/health',
+          path: '*',
```

Comment:
```suggestion
          path: '/',
```

Just because this is what we usually do, it's the same thing